app/ante/evm/utils_test.go

```diff
@@ -448,14 +448,19 @@ func (suite *AnteTestSuite) generateMultikeySignatures(signMode signing.SignMode
 	return signatures
 }
 
-// RegisterAccount creates an account with the keeper and populates the initial balance
-func (suite *AnteTestSuite) RegisterAccount(pubKey cryptotypes.PubKey, balance *big.Int) {
+// RegisterAccountIfNil creates an account with the keeper and populates the
+// initial balance if it is not already stored in the account keeper.
+func (suite *AnteTestSuite) RegisterAccountIfNil(pubKey cryptotypes.PubKey, balance *big.Int) {
```

Comment:
do we still need this change or we can roll it back?
Ideally, all tests should pass without the need of adjusting anything

Reply:
Tests are passing also without changing this method.But, the logic here is wrong because we are registering two times an already registered account, that's why I left the fix. I can also remove it but it should be fixed in a follow up PR.